### PR TITLE
feat: add "Examples" navigation link

### PR DESCRIPTION
**Pull request by molant**
Add the top navigation link "Examples" and a new page that explains how
to acquire Fiddle and run the examples available in the sidebar.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Partial work on #2

### CODE REVIEW

**Comment by molant** [comment]
This is how it looks like locally:

image

File: scripts/tasks/how-to-examples.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const replaceText = (selector, text) => {
+    const element = document.getElementById(selector)
+    if (element) element.innerText = text
+  }
+
+  for (const type of ['chrome', 'node', 'electron']) {
+    replaceText(`${type}-version`, process.versions[type])
+  }
+})
+```
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:29]: I have an equivalent section meant for the first page you see when you go to `electronjs.org/docs`:

https://github.com/electron/electron/blob/bf04a5f9f83d2af602946e24a2465c724121d77d/docs/tutorial/introduction.md#running-examples-with-electron-fiddle

**Review by erickzhao — COMMENTED** [review]
I wonder if instead of adding a whole doc on this, we should have a short admonition added to the existing `how-to` docs.

**Comment by molant** [comment]
what content do you think we should have as the landing of examples?

________________________________

File: scripts/tasks/how-to-examples.md (hunk 1 above)
Comment (molant) [reply scripts/tasks/how-to-examples.md:29]: @erickzhao what content do you think we should have as the landing of examples?

File: scripts/tasks/how-to-examples.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+slug: examples
+hide_title: false
+---
+
+# Examples
+
+In this section we have documented some of the most popular things developers do
+in their Electron applications. The easiest way to run the examples is using
+[Fiddle][fiddle]. To do this, first you will have to download and install the
+right version for your Operating System from [here][fiddle].
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:13]: I think linking the Fiddle link twice is a bit redundant here


```suggestion
In this section, we have collected a set of guides for common features
that you may want to implement in your Electron application. Each guide
contains a practical example in a minimal, self-contained example app.
The easiest way to run these examples is by downloading [Electron Fiddle][fiddle].
```

File: scripts/tasks/how-to-examples.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: 'Examples'
+description: 'Electron examples directly for you to use'
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:3]:
```suggestion
description: 'A set of examples for common Electron features'
```

File: scripts/tasks/how-to-examples.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+# Examples
+
+In this section we have documented some of the most popular things developers do
+in their Electron applications. The easiest way to run the examples is using
+[Fiddle][fiddle]. To do this, first you will have to download and install the
+right version for your Operating System from [here][fiddle].
+
+Once you have done this, you can press on the "Open in Fiddle" button that you
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:15]:
```suggestion
Once Fiddle is installed, you can press on the "Open in Fiddle" button that you
```

File: scripts/tasks/how-to-examples.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    replaceText(`${type}-version`, process.versions[type])
+  }
+})
+```
+
+## How to...?
+
+The following are all the "How to?" available at the moment (you can access them
+from the sidebar as well). If there is something that you would like to do that
+is not documented, please join our [discord server][] and let us know!
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:35]: Is there some kind of dynamic loading here since there's no list in the raw markdown?

Comment (molant) [reply scripts/tasks/how-to-examples.md:35]: I was thinking about doing it but then decided it shouldn't have to be part of the MVP (I have a couple other things coming at work :( ) and forgot to update the content.

Comment (molant) [reply scripts/tasks/how-to-examples.md]: I've removed the reference. We should definitely do it (I really like the example you provide from Gatsby) but I don't think it's necessary for the MVP. If you are OK with that I'll open an issue to track that effort.

Comment (erickzhao) [reply scripts/tasks/how-to-examples.md]: Sounds good to me!

File: scripts/tasks/how-to-examples.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+})
+```
+
+## How to...?
+
+You can find the full list of "How to?" in the sidebar. If there is
+something that you would like to do that is not documented, please join
+our [discord server][] and let us know!
+
+[discord server]: https://discord.com/invite/electron
```
Comment (erickzhao) [inline scripts/tasks/how-to-examples.md:37]:
```suggestion
our [Discord server][] and let us know!

[Discord server]: https://discord.com/invite/electron
```

**Review by erickzhao — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]

**Pull request merged**